direct/io-cassandra/src/main/java/cz/o2/proxima/direct/cassandra/CassandraRandomReader.java

```diff
@@ -130,14 +130,14 @@ public void scanWildcardAll(
           // by convention
           String name = wildcard.toAttributePrefix() + accessor.getConverter().asString(attribute);
 
-          Optional parsed = wildcard.getValueSerializer().deserialize(rowValue);
+          Optional<T> parsed = wildcard.getValueSerializer().deserialize(rowValue);
 
           if (parsed.isPresent()) {
             consumer.accept(
-                (KeyValue)
+                (KeyValue<T>)
                     KeyValue.of(
                         getEntityDescriptor(),
-                        (AttributeDescriptor) wildcard,
+                        (AttributeDescriptor<T>) wildcard,
```

Comment:
Looks like these casts can be removed now.

Reply:
Fixed